azure-pipelines.yml

```diff
@@ -11,17 +11,15 @@ pr:
     - dev
     - in-proc
     - release/4.*
-    - release/inproc6/4.*
-    - release/inproc8/4.*
+    - release/in-proc
 
 trigger:
   branches:
     include:
     - dev
     - in-proc
     - release/4.*
```

Comment:
we don't need `dev` or `release/4.*` here either, do we?

Reply:
We don't. Removed.

Reply:
I recommend keeping trigger section consistent between all branches. So leaving `dev` and `release/4.*` is probably a good idea.

Reply:
Okie, Added back "dev" and "release/4.*".